### PR TITLE
Allow to specify idxfile

### PR DESCRIPTION
**Pull request by ricfrank**
Add a new option -f / --file in order do specify a given idxfile. The syntax is:

idx -f myidxfile.php [args]

idx --file myidxfile.php [args]

### CODE REVIEW

**Comment by micheleorselli** [comment]
Thank @ricfrank for the contribution, Idephix needs **lot** of love ;-). I'm not totally convinced with this approach, there are some smells I'd like ti point out:

1. the ```-f/--file ``` parameter is handled **outside** idephix scope. This means that an ``` idx help ``` **will not** show the ```-f/--file ``` param. Fixing this is not trivial since the idephix class is instantiated **in** the idxfile, which in my opinion is quite an anti-pattern (see #41)  

2. tests should use the vfsStream lib as much as possibile and not relying on the real filesystem **or** if is not possibile, file used only for testing purposes should be moved in the test directory

**Comment by ricfrank** [comment]
@micheleorselli thanks for feedback.

1. i added --file (and -f) option to idx help https://github.com/ideatosrl/Idephix/commit/fc40593f23230daad52536de59c55c6511fef81e .
Is it a right way to do this ? I agree that is an anti-pattern to instantiate Idephix class in the idxfile, but at the moment i haven't a simple way to add this feature in an other way. We could discuss about  implementation on #41.

2. I haven't use vfsStream to mock filesystem because i don't interact with it. No file or directory is created and I thought that create a fake idxfile (in the test dir) was simpler and clearer. Is it ok ?

File: tests/Idephix/LoadCustomIdxFileTest.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public function testCallTaskWithParamsFromCustomIdxFile()
+    {
+        $output = shell_exec("php " . __DIR__ . "/../../bin/idx -f " .__DIR__ . "/../Test/idxfile_test.php idephix:test-params non per me");
+
+        $this->assertContains("non per me", $output);
+    }
+
+    public function testLoadDefaultIdxFile()
+    {
+        $this->markTestSkipped("Return binary value but should return a string");
```
Comment (micheleorselli) [inline tests/Idephix/LoadCustomIdxFileTest.php]: @ricfrank is it ok to skip that?

**Comment by micheleorselli** [comment]
@ricfrank 

1) I guess it's is a good compromise atm, let's go with this
2) yep got it

I think we could merge after fixing the skipped test

**Comment by ricfrank** [comment]
@micheleorselli can we merge ?

**Comment by micheleorselli** [comment]
:+1:  for me

**Pull request merged**